#! /bin/sh /usr/share/dpatch/dpatch-run
## 05-5.6-compat.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use x.6 SONAME as x.6.1 is ABI compatible with x.6

@DPATCH@
--- ACE_wrappers.orig/configure.ac
+++ ACE_wrappers/configure.ac
@@ -148,7 +148,7 @@
 ACE_MAJOR=$2
 ACE_MINOR=$3
 ACE_BETA=$4
-ACE_VERSION_NAME=ACE_VERSION
+ACE_VERSION_NAME=5.6
 
 AC_SUBST([ACE_MAJOR])
 AC_SUBST([ACE_MINOR])
--- ACE_wrappers.orig/TAO/MPC/config/taodefaults.mpb
+++ ACE_wrappers/TAO/MPC/config/taodefaults.mpb
@@ -7,7 +7,7 @@
   libpaths    += $(ACE_ROOT)/lib
 
   specific(automake) {
-    am_release = @TAO_VERSION_NAME@
+    am_release = @TAO_MAJOR@.@TAO_MINOR@
     includes += $(TAO_BUILDDIR)
     libpaths -= $(ACE_ROOT)/lib
   }
